app/cms-client/src/components/Pages/Bar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { loadComponent } from 'lib/Injector';
+import { Link, withRouter } from 'react-router-dom';
+
+const LeftAndMain = loadComponent('LeftAndMain');
+
+const Page = ({ match: { params: { parameterOne, parameterTwo } } }) => (
+  <LeftAndMain>
+    This is a Bar page. parameterOne={parameterOne} parameterTwo={parameterTwo}
+    <Link to="/admin/innovation">Back</Link>
+  </LeftAndMain>
+);
+
+export default withRouter(Page);
```

Comment:
`withRouter` adds some router context to the component and allows the Page to know what the url parameters are ... you could use a `useEffect` hook to fire off an http request based on the URL parameter.